src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```
так як в тебе тут саме продюсер бульдозерів краще одразу жорсткіше обмежитись і робити так щоб з методу get повертався тількі ліст Bulldozer, як в прикладі (аналогічно для інших продюсерів) 
![image](https://user-images.githubusercontent.com/58385506/119863440-3aba1900-bf22-11eb-89b6-31fcc1733f49.png)